src/view/form-edit-view.js

```diff
@@ -252,12 +266,15 @@ export default class FormEditView extends SmartView {
 
   #formSubmitHandler = (evt) => {
     evt.preventDefault();
+
+    if(this._data.type === '' || this._data.destination.name === '') {
```

Comment:
```suggestion
    if (this._data.type === '' || this._data.destination.name === '') {
```

Проверь форматирование по коду, что-то у тебя пробел плохо работать стал